elements/rh-spinner/rh-spinner.css

```
@@ -0,0 +1,81 @@
+:host {
+  display: inline-block;
+  aspect-ratio: 1 / 1;
+  text-align: center;
+  --rh-spinner--AnimationDuration: 1.2s;
+  --rh-spinner--StrokeWidth: 6px;
+  /*--rh-spinner--AnimationTimingFunction: ;*/
+}
+
+svg {
+  width: var(--rh-spinner--width, var(--rh-spinner--diameter, var(--rh-length-4xl, 64px)));
+  height: var(--rh-spinner--height, var(--rh-spinner--diameter, var(--rh-length-4xl, 64px)));
+  animation: rh-spinner-animation-rotate 
```

this animation name doesn't need a prefix, it's encapsulated in the shadow dom
```suggestion
  animation: rotate 
```

e.g. see [this demo](https://lit.dev/playground/#gist=88478ca5bd7ab1cb73ce0a24b487a013)

---

For some reason using `rotate` makes the blue stroke look slightly pixelated. Is there a benefit to using it vs a prefixed version?

---

the name of the animation shouldn't make a difference... could be there's something else going on here.

The benefit is readability, and fewer bytes over the wire